Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -1350,6 +1389,7 @@ static gboolean handleScroll(WebKitWebViewBase* webViewBase, double deltaX, doub
         // We only have hold events on touchpads, so assume one if we get no event.
         GdkInputSource source = device ? gdk_device_get_source(device) : GDK_SOURCE_TOUCHPAD;
 
+        bool isEnd = gdk_event_is_scroll_stop_event(event);
```

Comment:
Wait what? This event can be null, what happens then?

Reply:
Changed to handle event being null.

FWIW, event shouldn't be null here AFAICT and below it seems to assume the event is not null (calls gdk_event_get_modifier_state).